vendor/k8s.io/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -221,7 +222,10 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 			}
 
 			_, serverGroupsAndResources, err := crdClient.Discovery().ServerGroupsAndResources()
-			if err != nil {
+			switch {
+			case clientdiscovery.IsGroupDiscoveryFailedError(err):
+				return false, nil
```

Comment:
why not for every error?

Reply:
wanted to be conservative and see if we get any other errors... i believe we use loopback client, so there is little room for connection errors

Reply:
> wanted to be conservative and see if we get any other errors... i believe we use loopback client, so there is little room for connection errors

just ignore all errors (maybe log) and `return false, nil`.